learning/tour-of-beam/backend/docker-compose.yml

```diff
@@ -25,4 +25,4 @@ services:
       - DATASTORE_LISTEN_ADDRESS=0.0.0.0:8081
     ports:
       - "8081:8081"
-    command: --consistency=1.0
+    command: --consistency=1.0 --store-on-disk
```

Comment:
@eantyshev maybe we should try to set consistency to recommended 0.05 for development?

Reply:
@eantyshev  --store-on-disk is a enabled by default not sure we need it here

Reply:
> --store-on-disk is a enabled by default not sure we need it here
I believe better to set it explicitly, provided we have to define other cmd options, as the default may change in future

> maybe we should try to set consistency to recommended 0.05 for development?
This setup is about CI/CD, we don't want to introduce undefined behavior here